### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by miketully426**

### CODE REVIEW

**Comment by tpenn** [comment]
👍🏼

**Comment by joann37** [comment]
Looks good!

**Comment by julialoftin** [comment]
this is a comment

**Comment by shanika2009** [comment]
Pepsi

**Comment by haylied** [comment]
hi

**Comment by LunaLiu829** [comment]
Review done.

**Comment by sarahdao** [comment]
great!

**Comment by angelaengle67** [comment]
Doot

**Comment by tayloreisele** [comment]
🚀

**Comment by effab** [comment]
Looks good

**Comment by NavajoVulcan** [comment]
Daaaaaaaaaaaaaang, what a good pull req

**Comment by MuntoniFrancesca** [comment]
Hi!

**Comment by CleeMe** [comment]
Testing testing can I do this from my phone?

**Comment by AshleyRichardson** [comment]
HI!

**Comment by samanthakaymc** [comment]
Dr. Pepper

**Comment by monicasrinivasan27** [comment]
Good

**Comment by ekemlage** [comment]
Hello!

**Comment by tofu-rat** [comment]
hi mike!

**Comment by Cookieookie** [comment]
nice

**Comment by heiningermolly** [comment]
I'd love to have a quick rundown on how to clone a specific branch to test it before approving.

**Comment by milhousecvh** [comment]
🤘

File: src/pages/index.astro [hunk 1]
```diff
@@ -1,65 +1,11 @@
...
...7 159.985 255.147 153.581 247.358 153.581C239.569 153.581 232.669 159.985 230.306 169.408L167.19 364.254ZM160.869 530.172C160.877 530.18 160.885 530.187 160.894 530.195L160.867 530.181C160.868 530.178 160.868 530.175 160.869 530.172ZM136.218 411.348C124.476 450.467 132.698 504.458 160.869 530.172C160.997 529.696 161.125 529.242 161.248 528.804C161.502 527.907 161.737 527.073 161.917 526.233C165.446 509.895 178.754 499.52 195.577 500.01C211.969 500.487 220.67 508.765 223.202 527.254C224.141 534.12 224.23 541.131 224.319 548.105C224.328 548.834 224.337 549.563 224.347 550.291C224.563 566.098 228.657 580.707 237.264 593.914C245.413 606.426 256.108 615.943 270.749 622.478C270.593 621.952 270.463 621.508 270.35 621.126C270.045 620.086 269.872 619.499 269.685 618.911C258.909 585.935 266.668 563.266 295.344 543.933C298.254 541.971 301.187 540.041 304.12 538.112C310.591 533.854 317.059 529.599 323.279 525.007C345.88 508.329 360.09 486.327 363.431 457.844C364.805 446.148 363.781 434.657 359.848 423.275C358.176 424.287 356.587 425.295 355.042 426.275C351.744 428.366 348.647 430.33 345.382 431.934C303.466 452.507 259.152 455.053 214.03 448.245C184.802 443.834 156.584 436.019 136.218 411.348Z"
```
Comment (Rebekah314) [inline src/pages/index.astro:20]: That's a lot of numbers.

**Comment by bhavana1308** [comment]
Hello!

File: src/pages/index.astro [hunk 2]
```diff
@@ -1,65 +1,10 @@
...
-				title="Themes"
-				body="Explore a galaxy of community-built starter themes."
-			/>
-			<Card
-				href="https://astro.build/chat/"
-				title="Community"
-				body="Come say hi to our amazing Discord community. ❤️"
-			/>
-		</ul>
+		Do you want <b>Coke</b> or <i>Pepsi</i>?
```
Comment (erinsuzy) [inline src/pages/index.astro]: Have you considered RC instead?

**Comment by karengp0207** [comment]
Coke is the best!

File: src/pages/index.astro [hunk 3]
```diff
@@ -1,65 +1,11 @@
...
-				title="Themes"
-				body="Explore a galaxy of community-built starter themes."
-			/>
-			<Card
-				href="https://astro.build/chat/"
-				title="Community"
-				body="Come say hi to our amazing Discord community. ❤️"
-			/>
-		</ul>
+		<h1>Do you want <b>Coke</b> or <i>Pepsi</i>?</h1>
```
Comment (SamuelPhilMS) [inline src/pages/index.astro:7]: not sure about this one boss

File: src/pages/index.astro (hunk 1 above)
Comment (mendozacolonx) [inline src/pages/index.astro:20]: very interesting

**Comment by rtyUM333** [comment]
coke

File: src/pages/index.astro [hunk 4]
```diff
@@ -73,51 +19,4 @@ import Card from '../components/Card.astro';
 		font-size: 20px;
 		line-height: 1.6;
 	}
-	.astro-a {
```
Comment (sloanewolter) [inline src/pages/index.astro:76]: Test comment- Why is this called astro

**Comment by kitusha** [comment]
looks good

**Comment by LouisSloan** [comment]
（；´д｀）ゞ

**Comment by AnidaMalkic** [comment]
Theeeee comment!!!!

**Review by Rebekah314 — COMMENTED** [review]
Nice changes.

File: src/pages/index.astro [hunk 5]
```diff
@@ -1,65 +1,11 @@
 ---
 import Layout from '../layouts/Layout.astro';
-import Card from '../components/Card.astro';
 ---
 
-<Layout title="Welcome to Astro.">
+<Layout title="Welcome to Coke or Pepsi.">
 	<main>
```
Comment (JillianJaclyn) [inline src/pages/index.astro:6]: Coke

File: src/pages/index.astro [hunk 6]
```diff
@@ -1,65 +1,11 @@
...
-		<svg
-			class="astro-a"
```
Comment (BeesKnives) [inline src/pages/index.astro:9]: :)

File: src/pages/index.astro [hunk 7]
```diff
@@ -1,65 +1,10 @@
...
-					x1="247.358"
-					y1="0"
-					x2="247.358"
-					y2="622.479"
-					gradientUnits="userSpaceOnUse"
-				>
-					<stop stop-opacity="0.9"></stop>
-					<stop offset="1" stop-opacity="0.2"></stop>
-				</linearGradient>
-			</defs>
```
Comment (dveed) [inline src/pages/index.astro:34]: defs some code

**Comment by Haripriya92** [comment]
Code looks  great

**Review by ChristinaMartin13 — COMMENTED** [review]
🥇

**Comment by slmetoxen** [comment]
Looks great! Can't wait to see what's next.

**Comment by SarahLakey** [comment]
Pepsi

**Comment by Purpleater** [comment]
🦞

**Comment by jlynn1001** [comment]
Hi 👍🏻

**Comment by ahainline** [comment]
Gotta love deleting a bunch of lines of code

**Comment by dveed** [comment]
spicy meatball

**Comment by mollsvb14** [comment]
Coke Zero or nothing

File: src/pages/index.astro [hunk 8]
```diff
@@ -73,51 +18,4 @@ import Card from '../components/Card.astro';
 		font-size: 20px;
 		line-height: 1.6;
 	}
-	.astro-a {
-		position: absolute;
-		top: -32px;
```
Comment (NavajoVulcan) [inline src/pages/index.astro:78]: Parkour

**Comment by specialtas** [comment]
yo so great!

File: src/pages/index.astro [hunk 9]
```diff
@@ -1,65 +1,11 @@
...
-			<defs>
-				<linearGradient
-					id="paint0_linear_1805_24383"
-					x1="247.358"
-					y1="0"
-					x2="247.358"
-					y2="622.479"
-					gradientUnits="userSpaceOnUse"
-				>
-					<stop stop-opacity="0.9"></stop>
```
Comment (LouisSloan) [inline src/pages/index.astro:31]: （；´д｀）ゞ

**Comment by Ssmarie14** [comment]
Noiceeeee

**Pull request merged**